### PR TITLE
Wrap ask errors from WSCWA to avoid ClassCastException.

### PR DESCRIPTION
**Pull request by kshakir**

### CODE REVIEW

**Comment by kshakir** [comment]
Hotfix version

**Comment by Horneth** [comment]
Maybe add an entry in the changelog, even just
```
## 33.1 Release Notes

### Bug fixes
```

So that something shows up in the release in github (if you do make a release)

**Comment by mcovarr** [comment]
CHANGELOG.md looks good 😄

File: engine/src/main/scala/cromwell/engine/workflow/workflowstore/WorkflowStoreCoordinatedWriteActor.scala [hunk 1]
```diff
@@ -22,7 +22,7 @@ class WorkflowStoreCoordinatedWriteActor(workflowStore: WorkflowStore) extends A
   def run[A](future: Future[A]): Unit = {
     val result = Try(Await.result(future, Timeout)) match {
       case Success(s) => s
-      case f: Failure[_] => f
+      case f: Failure[_] => Status.Failure(f.exception)
```
Comment (cjllanwarne) [inline engine/src/main/scala/cromwell/engine/workflow/workflowstore/WorkflowStoreCoordinatedWriteActor.scala]: 🤦‍♂️ 
And ToL: Maybe `case Failure(e) => Status.Failure(e)` is a tiny bit neater?

Comment (kshakir) [reply engine/src/main/scala/cromwell/engine/workflow/workflowstore/WorkflowStoreCoordinatedWriteActor.scala]: Hehe, very good point. But this tiny fix has been a 💩-show. Will update you IRL if you care. Merging but PRs accepted. 😉

**Pull request merged**